assets/js/screens/Collection/List.jsx

```diff
@@ -1,17 +1,27 @@
 import React, { useState, useEffect, useRef } from "react";
 import CollectionListRow from "../../components/Collection/ListRow";
 import { useQuery } from "@apollo/react-hooks";
-import { GET_COLLECTIONS } from "../../components/Collection/collection.query";
+import {
+  GET_COLLECTIONS,
+  DELETE_COLLECTION
+} from "../../components/Collection/collection.query";
 import Error from "../../components/UI/Error";
 import Loading from "../../components/UI/Loading";
-import { Link } from "react-router-dom";
-import debounce from "lodash.debounce";
+import { Link, useHistory } from "react-router-dom";
+import { useMutation } from "@apollo/react-hooks";
+import { useToasts } from "react-toast-notifications";
+
 import Layout from "../Layout";
+import UIModalDelete from "../../components/UI/Modal/Delete";
 
 const ScreensCollectionList = () => {
+  const { addToast } = useToasts();
   const { data, loading, error } = useQuery(GET_COLLECTIONS);
   const [filteredCollections, setFilteredCollections] = useState([]);
-  const [searchValue, setSearchValue] = useState("");
+  const history = useHistory();
+  const [deleteCollectionId, setDeleteCollectionId] = useState(null);
```

Comment:
No need to define a `null` value here.  Just use `useState()` to keep things clean.

Reply:
> All in all this looks really good. A few requested changes...
> 
> After you make and commit your changes, make sure to pull down the latest `deploy/staging` branch, then switch back to your feature branch `513-verify-collections-crud` and rebase on top of `deploy/staging`. The commands are:
> 
> ```
> git checkout deploy/staging
> git fetch
> git status (to see if you're up to date)
> git pull (if necessary)
> git checkout 513-verify-collections-crud
> git rebase deploy/staging
> ```
> 
> I'd like to get a situation eventually where there's a merge conflict, so we can walk through how to handle those. If you don't see any errors, then you can push up your changes.
> 
> Ideally we like to only include one commit per PR, so we can walk through how to squash your commits into one (if you made an extra commit for the requested changes).
> 
> Any questions, just ping me and we'll work through it.

Can I use this for combining commits ?
`git reset HEAD~
git add .
git commit -C ORIG_HEAD
git push -f origin`

Reply:
Yea for sure.  If going that route, go ahead and make your adjustments, then when it's all working as you like, run the above commands you mentioned.  When you force push your feature branch back up, `git push -f origin`, you may have to include your branch title as the final argument if you don't have your remote feature branch defined as an upstream branch.